components/search/src/filter.rs

```diff
@@ -214,16 +233,30 @@ impl Filter for Vec<JSONSearchConfigurationRecords> {
     }
 }
 
+fn apply_overrides(
```

Comment:
I think this could possibly be moved to before it is used, as I think that helps readability.